pkgs/applications/misc/gopass-ui/node-dependencies.nix

```
@@ -0,0 +1,184 @@
+{ pkgs ? import <nixpkgs> { inherit system; }
+, system ? builtins.currentSystem
+, gopass-ui-src ? ./.
+}:
+
+with pkgs;
+
+let
+  nodePackages = import ./node-composition.nix {
+    inherit system;
+    nodejs = nodejs-12_x;
+    pkgs = {
+      inherit fetchurl fetchgit;
+      inherit stdenv python2 utillinux runCommand writeTextFile;
+      inherit darwin;
+    };
+  };
+
+  electron = electron_7;
+  electron-chromedriver = if pkgs ? electron-chromedriver_3
+    then electron-chromedriver_3
+    else null;
+  # This file is written by `node_modules/electron/install.js::extractFile`
+  # and seems to be mandatory for the package to recognise that the bin dist
+  # has been properly installed.
+  electronPathTxtFile = writeTextFile (
+    let
+      platform = lib.lists.last (builtins.split "-" system);
+      # Should match what is returned by `node_modules/electron/install.js::getPlatformPath`.
+      content = ({
+        "darwin" = "Electron.app/Contents/MacOS/Electron";
+        "freebsd" = "electron";
+        "linux" = "electron";
+        "win32" = "electron.exe";
+      })."${platform}";
+    in
+  {
+    name = "path.txt";
+    text = content;
+  });
+
+  buildInputs = [
+    # Patching various `package.json` files.
+    jq
+  ] ++ stdenv.lib.optionals stdenv.isDarwin [ xcbuild darwin.DarwinTools ];
```

I've managed to build on darwin with this fix:
```suggestion
  ] ++ stdenv.lib.optionals stdenv.isDarwin [ xcbuild darwin.DarwinTools darwin.apple_sdk.frameworks.CoreServices ];
```

---

Alright, added this.